qtcreator.pri

```diff
@@ -131,7 +131,10 @@ osx {
     isEmpty(IDE_OUTPUT_PATH): IDE_OUTPUT_PATH = $$IDE_BUILD_TREE
 
     IDE_LIBRARY_PATH = $$IDE_OUTPUT_PATH/$$IDE_LIBRARY_BASENAME/qtcreator
-    IDE_PLUGIN_PATH  = $$IDE_LIBRARY_PATH/plugins
+    os2: \
```

Comment:
Hmm, why putting plugins into /bin/plugins instead of /plugins as on all other systems?

Reply:
in my manjaro build the plugins are under bin